repo/packages/Q/quobyte/2/command.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
@casusbelli In that case, I think we can add the native client in next version. Can you test the package that it works with this python client?